components/TokensGrid.tsx

```diff
@@ -28,23 +17,15 @@ type Props = {
 }
 
 const TokensGrid: FC<Props> = ({ tokens, viewRef, collectionImage }) => {
-  const [cartTokens, setCartTokens] = useRecoilState(recoilCartTokens)
-  const cartCurrency = useRecoilValue(getCartCurrency)
-  const tokensMap = useRecoilValue(getTokensMap)
-  const { data: signer } = useSigner()
-  const { chain: activeChain } = useNetwork()
   const { data, mutate } = tokens
-  const account = useAccount()
-  const reservoirClient = useReservoirClient()
+  const [cartTokens] = useRecoilState(recoilCartTokens)
```

Comment:
Do we really need this here? Seems like the only use for it is type inference on line 23.